_posts/2022-01-21-safe-releases.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Reducing thrash and increasing throughput by focusing on elegant releases
+author_avatar: https://avatars.githubusercontent.com/u/8116516?v=4
+author_name: Xavier Hocquet
+author_github_username: xhocquet
+---
+
+Continuous integration and deployment is fundamental to a high-performing technology business as set forth by Kim et al. Deploying frequently results in smaller releases, more frequent updates for your users, quicker feedback from real users, and a tighter feedback loop between business decisions and shipped products.
```

Comment:
@xhocquet , it is possible to link to a paper or something concerete by Kim et al? It's always useful to have that reference for the reader.

Reply:
If you think it's useful, calling out CI/CD as a core tenent for agile development would be good here. 

This is how NewRelic summarizes it - "Associated with both Agile and DevOps, they all work together to allow businesses to leverage automation in order to develop, build, test, and deploy higher-quality code more rapidly."

https://newrelic.com/blog/best-practices/continuous-delivery-continuous-deployment-continuous-integration#:~:text=Continuous%20integration%2C%20delivery%2C%20and%20deployment,higher%2Dquality%20code%20more%20rapidly.